news/2025/2025-04-24-osutaiko-world-cup-finals-recap.md

```diff
@@ -0,0 +1,91 @@
+
+After staving off a huge second half comeback to take Chile's second pick of [cosMo@BousouP feat. Hatsune Miku - DeroDeroDeroDe](https://osu.ppy.sh/beatmapsets/2317287#taiko/4962143) (HD2), Taiwan finally broke the break streak with a dominant win on [hugepulse - Kaitei Gensou Toshi](https://osu.ppy.sh/beatmapsets/1955875#taiko/5069532) (NM7), with ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087) narrowly missing out on a 99.5% full combo that would give the winner’s bracket a run for their money! Chile held on with three amazing HDHR scores on [Umeboshi Chazuke - Galaxy in Toybox](https://osu.ppy.sh/beatmapsets/559156#taiko/1182564) (FM1) to make it 2-3, Taiwan printed a point from the polyrhythmic [KERO - Color Printer](https://osu.ppy.sh/beatmapsets/2167213#taiko/4572334) (HR1), and Chile kept up on [KNOWER - Time Traveler](https://osu.ppy.sh/beatmapsets/2354840#taiko/5070071) (DT1) by way of not one but three full-combo near misses on one of the week's toughest maps.
+
+Now up by a score of 3-4, Taiwan seemingly found an extra gear that Chile couldn't quite seem to match. Two full combos on [Spire of Lazarus - Big Hat Logan](https://osu.ppy.sh/beatmapsets/2354620#taiko/5069531) (NM5) with an unreal 3x100 from ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087), three scores with one combined miss on the highly stressful [Amanogawa Converter - Never play me](https://osu.ppy.sh/beatmapsets/2354618#taiko/5069528) (NM4) for one more break point, and one tightly contested, high-accuracy battle on [Rahatt - Gunoi Chop](https://osu.ppy.sh/beatmapsets/2354610#taiko/5069511) (NM1) later, team Taiwan had done it.
+
+With a huge 7-3 win over a terrifying opponent in Chile, who still deserve huge congratulations for their best finish ever at 4th place, Taiwan came out on top of the brutal Finals lower bracket, advancing to the Grand Finals.
+
+## Score highlights
+
+In a later stage like the Finals, there is no avoiding scores that seem absoultely inhumane to a spectator. For players that have never played osu!taiko themselves, and even for players that are longstanding members of the community, some of those scores are incomprehensible.
+
+- One of those scores last weekend, was the unreal double S score from ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196) on the very experimental and difficult NM2 pick, [Terminal 11 - Echoes](https://osu.ppy.sh/beatmapsets/2354615#taiko/5069524). The Japanese player has outscored every other player in their match against South Korea with zero 100s and outstanding gameplay throughout our brand new osu! original.
```

Comment:
the last bit sounds extra when the focus is otherwise on the score

```suggestion
- One of those scores last weekend was the unreal double S score from ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196) on the very experimental and difficult NM2 pick, [Terminal 11 - Echoes](https://osu.ppy.sh/beatmapsets/2354615#taiko/5069524). The Japanese player has outscored every other player in their match against South Korea with zero 100s and outstanding gameplay.
```